test/common/stats/symbol_table_speed_test.cc

```diff
@@ -17,46 +17,48 @@
 
 // NOLINTNEXTLINE(readability-identifier-naming)
 static void BM_CreateRace(benchmark::State& state) {
-  Envoy::Thread::ThreadFactory& thread_factory = Envoy::Thread::threadFactoryForTest();
+  for (auto _ : state) {
```

Comment:
was this change in the structure of this test (beyond just using the framework) motivated by something in particular?

Reply:
Yes, the original benchmark crashes.

Reply:
strange; it did not do that when I checked it in, but moving forward :)